### ISSUE TITLE
Unable to re run the app after a crash due to full hard drive

### ISSUE THREAD

**Issue by Buisson** [xypwn/southpark-downloader-ui#24]
Hello, I was using your app and everything was ok. But once my hardrive was full the app has crash.

After that I made some clean to have enough space, but now I can't run the app anymore, I tried to restart my computer, clear my temp files re download your app, but anything was working.

Where are the files / logfiles to see what happen and how to fix this ?

I am on Windows 10.

Thanks !

**Comment by xypwn** [comment]
Sure, that would be great :) You should be able to just upload the contents as an archive.

**Comment by xypwn** [comment]
Thanks! I'd agree it's probably downloads.json. This is likely a bug in Fyne (the GUI app library I used), so I don't know if I'll actually fix it. Also, my guess is the bug needs very specific timing to occur. If I find the time, I'll look at it.

You have a nice day too :)

PS: Did you get it to work after again deleting the folder?

**Comment by Buisson** [comment]
Yes it is running, but now the list of files that have been download was lost in the process.

There is a way to synchronise my current directory to populate the downloads.json with the list of episodes that was already download ? Maybe a script ?

Thank you so much for the support again !

**Comment by xypwn** [comment]
Unfortunately there's not currently a way to do this (this is what downloads.json was supposed to do). I hope you didn't lose too many downloads.

**Comment by Buisson** [comment]
All of them :). I have the files but the app doesn't see them and say that it is actually downloaded.

But not a problem, maybe I'll make a script to populate the download.json to sync the app :).

**Comment by xypwn** [comment]
Good luck! I'm sorry that happened 🥲
